addon/_locales/zh_TW/long_description.html

```diff
@@ -0,0 +1,9 @@
+當你點選單按鈕時，這個套件會<strong>偵測</strong>目前頁面的開放街圖參數，並且會<strong>產生</strong>相關地圖與對應參數清單。
+
+認可的參數包括:
+<ul>
+<li>地圖經緯度 (縮放、經度、緯度)</li>
+<li>OSM 元素 (節點 id、路徑 id、關聯 id 與變更組合 id)</li>
```

Comment:
Is it common not to translate "id" ?

If it's not clear, "id" refers to "identificator". 
For example, the link https://www.openstreetmap.org/node/2706360406 points to a node with id 2706360406